src/TesApi.Web/DatabaseOverloadedExceptionMiddleware.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace TesApi.Web
```

Comment:
I'd suggest moving this file to the `Controllers` folder, since that's the only context in which this code will ever operate.